prow/plugins/plugins.go

```diff
@@ -178,7 +178,7 @@ type Configuration struct {
 	SigMention           SigMention             `json:"sigmention,omitempty"`
 	Size                 *Size                  `json:"size,omitempty"`
 	Triggers             []Trigger              `json:"triggers,omitempty"`
-	Welcome              Welcome                `json:"welcome,omitempty"`
```

Comment:
If we wanted to be extra special we could keep this field, and then append it to WelcomeMessage if it != "" and post a log warning that this field is deprecated and will go away in 2019 or whenever.

Totally fine with not bothering to do this too.

If we did it it would probably involve extending the Load() method (either adding a new validate or maybe the defaults?): https://github.com/kubernetes/test-infra/blob/9df33c0c2556563730afdf8f1440b266dd06e8d5/prow/plugins/plugins.go#L687-L695